### PR TITLE
[New] Add ammaps w/ git auto-update and update/enrich package.json

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10958 #10285

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/amcharts/ammap3
 * Official website (optional, not the repository): https://www.amcharts.com/
 * License and its reference: [Custom](https://github.com/amcharts/ammap3#license) & [Commercial License](https://github.com/amcharts/ammap3#commercial-license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 12
   - Count of stars: 64
   - Count of forks: 36

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
1. The source issue is labeled high priority though it has insufficient popularity.
2. The library had existed on CDNJS, just add auto-update config and add its latest version.
3. Though it's not a new library we are going to host, I still `cc` the author because he opened the issue, I think we will want to receive notifications.
4. `license` part is based on [to be filled out] and https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/jquery-infinitescroll/package.json#L36
5. About the auto-update config
    1. Use `!(plugins|exporting)/**/*` for all files under other folders. 
        1. We don't need the files under `plugin` after v3.14.0  because they are from the libraries including [dataloader](https://github.com/amcharts/dataloader), [export](https://github.com/amcharts/export) and [responsive](https://github.com/amcharts/responsive). **I will add them later**.
        2. We don't need the files under `exporting` except `exporting/amexport.js` before v3.13.3 because they are from the libraries which had been hosted by CDNJS. They include canvg.js - [canvg](https://cdnjs.com/libraries/canvg), ...
I have not finished the comments... Will finish ASAP.

**Comment by sufuf3** [comment]
The order of the commits needs to be changed.
First, update the metadata; Second add the new version.
you could refer to https://github.com/cdnjs/cdnjs/pull/11551#pullrequestreview-59526021

**Comment by extend1994** [comment]
@sufuf3 New version (v3.21.6) is added after metadata update.

**Comment by sufuf3** [comment]
I think we don't need to cc the author, what do you think?
Because we are already hosting this lib.
```
Add ammap@3.21.6 w/ git auto-update
close #10285, cc @martynasma
```
-->
```
Add ammap@3.21.6 w/ git auto-update
close #10285
```

**Comment by extend1994** [comment]
@sufuf3 Please see https://github.com/cdnjs/cdnjs/pull/11873#issuecomment-331842906 point 3
> Though it's not a new library we are going to host, I still `cc` the author because he opened the issue, I think we will want to receive notifications.

**Comment by sufuf3** [comment]
@extend1994 I think the lib's name is ammaps
The commit message `Add ammap@3.21.6 w/ git auto-update` need to be changed to `Add ammap**s**@3.21.6 w/ git auto-update`
What do you think?

**Comment by extend1994** [comment]
@sufuf3 Thanks. Just update it. Please help review it.

**Comment by maruilian11** [comment]
@extend1994 I think we can remove the word `missing` in `Add missing minified files for ammaps@3.13.0` because it's not missing.

**Comment by maruilian11** [comment]
https://github.com/amcharts/ammap3/blob/master/ammap/plugins/export/package.json
`export` seems not on cdnjs, maybe we should open a PR.

**Comment by extend1994** [comment]
@maruilian11 I think they are missing from cdnjs?

**Comment by sufuf3** [comment]
Sorry for not notice.
1. 
```
Update and enrich package.json of ammaps, cc #10285
    
Update description, keywords, homepage, license & add author, homepage,
repository based on the package.json from upstream and official website.
```
You updated description, keywords, homepage, license and added author, repository.
So `Update description, keywords, homepage, license & add author, **homepage**,` homepage need to be remove, what do you think?

2. 
```
Add missing minified files for ammaps@3.13.0
    
Minify the files using web-minify-helper. close #10958
```
I think @maruilian11 's suggestion is good.
How about mention the missing files in detail commit?
Such as 
```
Add minified files for ammaps@3.13.0
    
Minify the files using web-minify-helper because the files are missing
from CDNJS. close #10958
```

What do you think?
Except for these part, I thinks this PR is good to me.
Thanks.

**Comment by maruilian11** [comment]
I think missing means it was there because some reasons, it's gone.

**Comment by extend1994** [comment]
Commit msg updated. @sufuf3 @maruilian11 Please help review these, thanks!

**Comment by PeterDaveHello** [comment]
Let @maruilian11 request the review from me once it's approved by her.

**Comment by sufuf3** [comment]
@PeterDaveHello Should this PR need to be waited for @maruilian11's review?
Would you please take a look?

Many thanks.

**Comment by sashberd** [comment]
@PeterDaveHello This PR is waiting FYR

**Comment by sashberd** [comment]
@PeterDaveHello ping

**Comment by PeterDaveHello** [comment]
@sashberd we are handling those PRs right now, give me more time ;)

File: ajax/libs/ammaps/package.json [hunk 1]
```diff
@@ -1,14 +1,39 @@
 {
   "name": "ammaps",
   "filename": "ammap.js",
-  "version": "3.13.0",
```
Comment (MattIPv4) [inline ajax/libs/ammaps/package.json]: Version should not be removed here, as this is the current version that we have on the CDN.

Comment (MattIPv4) [inline ajax/libs/ammaps/package.json]: Nothing has changed since last review, this version removal still needs to be reverted.

**Pull request merged**